frontend/public/components/app.jsx

```diff
@@ -153,29 +154,33 @@ class App_ extends React.PureComponent {
     const content = (
       <>
         <Helmet titleTemplate={`%s · ${productName}`} defaultTitle={productName} />
-        <QuickStartDrawer id="app-container">
-          <ConsoleNotifier location="BannerTop" />
-          <Page
-            header={<Masthead onNavToggle={this._onNavToggle} />}
-            sidebar={
-              <Navigation
-                isNavOpen={isNavOpen}
-                onNavSelect={this._onNavSelect}
-                onPerspectiveSelected={this._onNavSelect}
-              />
-            }
-          >
-            <ConnectedNotificationDrawer
-              isDesktop={isDrawerInline}
-              onDrawerChange={this._onNotificationDrawerToggle}
-            >
-              <AppContents />
-            </ConnectedNotificationDrawer>
-          </Page>
-          <CloudShell />
-          <GuidedTour />
-          <ConsoleNotifier location="BannerBottom" />
-        </QuickStartDrawer>
+        <QuickStartsLoader>
```

Comment:
Move `QuickStartsLoader` into `QuickStartsDrawer`. use an intermediate component in between if needed.

Reply:
moved inside drawer and separated the panel component